ce-kafka/README.md

```diff
@@ -32,4 +32,4 @@ Start by reading our guidelines on contributing to this project found here.
 
 ## License
 
-This Docker image is licensed under the Apache 2 license. For more information on the licenses for each of the individual Confluent Platform components packaged in this image, please refer to the respective [Confluent Platform documentation](https://docs.confluent.io/platform/current/installation/docker/image-reference.html).
+Usage of this image is subject to the license terms of the software contained within. Please refer to Confluent's Docker image software license terms [here](https://docs.confluent.io/platform/current/installation/docker/image-reference.html).
```

Comment:
@prabhamanepalli what do you think about this wording? Slightly modified form of the original suggestion. I'll leave this as a draft PR until the wording gets a green light